### PR TITLE
Implemented Tasks

### PR DESCRIPTION
**Pull request by KarinaKrinitskaya**

### CODE REVIEW

File: CSharpCollections1/CSharpCollections1/Citizen.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        }
+
+        #endregion
+
+        #region Methods
+
+        public bool CanVote()
+        {
+            int legalAge = 18;
+            return GetAge() > legalAge ? true : false;
```
Comment (MariaEroshevich) [inline CSharpCollections1/CSharpCollections1/Citizen.cs]: I think it should be >= legalAge

File: CSharpCollections1/CSharpCollections1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+foreach (var c in citizens)
+{
+    if (c.CanVote())
+    {
+        voters.Add(c);
+    }
+}
+
+voters.Sort();
```
Comment (MariaEroshevich) [inline CSharpCollections1/CSharpCollections1/Program.cs:127]: I don't see the code to create new lists of voters living at the given address

**Review by MariaEroshevich — CHANGES_REQUESTED** [review]

File: CSharpCollections1/CSharpCollections1/Citizen.cs (hunk 1 above)
Comment (KarinaKrinitskaya) [reply CSharpCollections1/CSharpCollections1/Citizen.cs]: Fixed

File: CSharpCollections1/CSharpCollections1/Program.cs (hunk 2 above)
Comment (KarinaKrinitskaya) [reply CSharpCollections1/CSharpCollections1/Program.cs:127]: fixed:)

**Review by MariaEroshevich — APPROVED** [review]

**Pull request merged**